README.md

```diff
@@ -7,51 +7,53 @@ This project was built using only built in web services (ie no NPM packages besi
 
 ### 🎯 Selection API
-- Find (and the unimplemented Replace)
-- Column, Col positions
-- Copy, Cut, Paste, Select all
-- Limitations
-  - Gets really tricky with `shadowRoot`. Supported in chromium but not everywhere
-
-#### 🗄️ Local Storage
-- Saving state
-  - Notepad text
-  - Settings
-  - Last search string in the find/replace bar
-- Limitations
-  - Making sure this isn't bogging down someones browser when things get LARGE (is this real?)
+
+Notepad has a set of "Edit" functions that do an array of different text insertion, deletion and searching. In order to figure out what portion of the editor the user is targeting, the Selection API gives the "range" of highlighted text or the singular cursor position. This allows the developer know where to "Time/Date" or where to paste text. For find/replace, we searched beginning and end of the desired substrings within the editor and then used the Selection API to create a range (which highlights the text) at the given start and end point. Once highlighted, we can control whatever text belongs in the range to replace, cut, copy or delete.
+
+The framework being used caused for some hiccups when getting started with this API. Lit is a web components frame work where you can create these smaller building block components and bring them together to create a full webpage and when nested create `shadowRoots` that have thier own scope (per component). This causes functions that are normally called like `window.getSelection()` to not return what the documentation says you should expect because things are buried within shadowRoots. In chromium, `shadowRoot.getSelection()` is supported and due to this, the Notepad PWA will only work in chromium environments since find/replace and all the edit functions are core to the experience.
+
+### 🗄️ Local Storage
+
+This PWA uses local storage to save a lot of state variables and settings. This includes saving the last entered text into the editor (even if the user doesn't save) to make sure the PWA reopens to this text next time the launch the app, also ensuring their settings are persistent across mutiple sessions of using the app.
+
+We began the work of adding tabs to the Notepad PWA to get closer to the Native app, however, we ran out of time and wanted to prioritize discovery of the gap between Native and PWA rather than how well can we code a Notepad PWA. Introducing tabs adds a layer of complexity with using local storage because now there are somethings that need to persist across all of Notepad (Settings or the last thing the user searched) and others that are specific to the Notepad instance or tab (the text in the editor, the file handler).
+
+While using local storage is usually pretty light on processing power, if you are going to be reading and writing to local storage often, it can begin to slow down your App, though we never reached that point with the Notepad PWA.
```

Comment:
```suggestion
While using local storage is usually pretty light on processing power, if you are going to be reading and writing to local storage often, it can begin to slow down your app, though we never reached that point with the Notepad PWA.
```